web/js/layers/layers.options.js

```diff
@@ -1,7 +1,14 @@
-var wv = wv || {};
-wv.layers = wv.layers || {};
-
-wv.layers.options = wv.layers.options || function (config, models, layer) {
+import $ from 'jquery';
+import each from 'lodash/each';
+import indexOf from 'lodash/indexOf';
+import isEqual from 'lodash/isEqual';
+import isUndefined from 'lodash/isUndefined';
+import parseInt from 'lodash/parseInt';
+import util from '../util/util';
+import wvui from '../ui/ui';
+import palettes from '../palettes/palettes';
+
+export function layersOptions(config, models, layer) {
```

Comment:
What do you think about keeping the name `options`, importing this into the `layers` parent, and attaching it to the object that exports?